wolfcrypt/src/ecc.c

```diff
@@ -7014,7 +7032,31 @@ int wc_ecc_verify_hash_ex(mp_int *r, mp_int *s, const byte* hash,
    err = silabs_ecc_verify_hash(&sigRS[0], keySz*2,
                                 hash, hashlen,
                                 res, key);
+#elif defined(WOLFSSL_SE050)
+    /* Used when following a hardware sign operation */
+    /*if (key->keyId != 0){
+        word32 outLen = key->sigLen;
+        err = StoreECC_DSA_Sig(sigRS, &outLen, r, s);
+        if (err != 0)
+            return err;
+    }*/
+    /* Used when not following a hardware sign operation */
+    /*else if (key->keyId == 0) {*/
+
+    int rLeadingZero = mp_leading_bit(r);
```

Comment:
The variable declarations all need to be at top of function or wrapped in braces `{}`.